resultstore_client.py

```diff
@@ -14,8 +14,7 @@
                                  get_default_configuration, gen_new_uuid,
                                  get_parent, get_default_target,
                                  get_default_configured_target,
-                                 get_default_action,
-                                 get_name)
+                                 get_default_action, get_name)
```

Comment:
what linter are you using? can we use this one: http://google.github.io/styleguide/pyguide.html

Reply:
I was using yapf suggested in the url